docs/naps/5-new-logo.md

```diff
@@ -0,0 +1,190 @@
+bigger circle. In this way, the island is defined by four circles of increasing
+radius, with each increase a factor of $\phi$:
+
+![napari-logo-3-with-guides](../images/napari-logo-3-with-guides.png)
+
+Additionally, the background of the logo is a
+[squircle](https://www.johndcook.com/blog/2018/02/13/squircle-curvature/) of
+degree 5, as Apple [uses](https://stackoverflow.com/a/47342594/224254) for iOS
+and macOS icons currently.
+
+Overall, the new icon still keeps the "napari" character, while looking more
+modern, friendlier, and more abstract. This is an advantage for those who don't
+know the origin of the logo, as they would still be able to appreciate the
+regularity of the shapes used.
+
+Additionally, the logo looks quite a bit like a dividing budding yeast *(S.
+cerevisiae)*, rather than an amorphous anonymous cell, which anyone with a
+background in biology would appreciate!
+
+Here is the proposed logo together with the old one:
+
+![napari-logo-comparison](../images/napari-logo-old-new.png)
+
+The new logo at 100% scale:
+
+![napari-logo-3](../images/napari-logo-3.png)
+
+## Related Work
+
+N/A
+
+## Implementation
+
+This is the fun bit. 😃
+
+The logo proposed above is generated *entirely within napari*. It makes use of:
+
+- A Shapes layer for each element: the background, the sand, the rainforest,
+  and the lagoon.
+- Scaling, translation, and rotation on each layer: the arithmetic for finding
+  the exact centers and contact points of all the circles is much easier with
+  the origin at the center of the north-west circle, a radius of
+  $1/\phi = \phi - 1$, and the island aligned along the 0th (vertical) axis.
+  Similarly, the squircle is easier to draw with the origin at the center. I
+  then apply scale, translation and rotation so that the island is at 45° and
+  the logo fills the space in [[0, 1024], [0, 1024]].
+- NumPy for computation of all the lines and shapes.
+
+Additionally, napari was used extensively during the development, to debug
+errors in calculating the contact points and so on. 😅
+
+It's very fun to work with powers of $\phi$: $1/\phi = \phi - 1$, which implies
+$1/\phi^2 = (\phi - 1)/\phi = 1 - 1/\phi = 2 - \phi$, and so on. Similarly,
+$\phi^2 = 1 + \phi$, which again lets you reduce all polynomials of $\phi$ to
+degree 1.
+
+The original implementation is at https://github.com/jni/new-napari-logo, but
+if this NAP is accepted, it would add the logo-generating script as a gallery
+example.
+
+Another bit of mathematical fun: Veritasium calls $\phi$ "the fiveiest number"
+(in [this fantastic video](https://youtu.be/48sCx-wBs34)): it is the ratio of
+the diagonal of a regular pentagon to its side, and it can be written as
+$\phi = 0.5 + 0.5 \times 5^{0.5}$! So it is a happy not-quite-coincidence that
+this is NAP-5. 😉
+
+## Backward Compatibility
+
+This NAP does not raise backwards compatibility concerns.
+
+## Future Work
+
+As mentioned above, this NAP is about the shapes of the logo, not the colors.
```

Comment:
I agree it could make sense to separate colors from shapes. However,  I think it would be useful to list the RGB colors used for the different parts of the logo for people developing other materials (e.g., posters/slides).

Reply:
I mean, they'll have the code... 😉

Reply:
People should absolutely just be able to download the logos from a page on napari.org, not be forced to generate them themselves @jni

Here's one example (I think this could be better if it also included a theme for powerpoint and google slides): https://www.dask.org/brand-guide